src/ucp/wireup/wireup_cm.c

```diff
@@ -1022,7 +1022,9 @@ ucs_status_t ucp_ep_client_cm_connect_start(ucp_ep_h ucp_ep,
 
     status = ucp_ep_client_cm_create_uct_ep(ucp_ep);
     if (status != UCS_OK) {
-        return status;
+        if (!ucp_cm_client_try_fallback_cms(ucp_ep)) {
+            ucp_ep_set_failed_schedule(ucp_ep, ucp_ep_get_cm_lane(ucp_ep), status);
```

Comment:
add ucs_error() message

Reply:
for ucp_cm_client_try_fallback_cms(), the only error path in the function already had ucs_diag(), does it need ucs_error() here?

Reply:
ok, no need ucs_error then